lib/pointfree.doc.md

```diff
@@ -0,0 +1,37 @@
+# pointfree
+
+Pointfree takes a string and optional integer. The string should be a method name on an existing object. The integer specifies the arity of the method (the number of expected arguments) and defaults to 1. It returns a curried function that accepts the specified number of arguments followed by the object containing the method. After all parameters are received it will call the method with the provided arguments and return the result.
+
+```javascript
+
+import { pointfree, curry } from 'lb-f'
+
+const map = pointfree('map')
+const reduce = pointfree('reduce', 2)
+
+const array = [1, 2, 3]
+const add = curry( (a, b) => a + b )
+const add1 = add(1)
+
+map(add1, array) // [2, 3, 4]
+reduce(add, 0, array) // 6
+
+```
+
+## In the real world
+
+Pointfree can be used to easily convert easily convert methods on arrays or other objects to a point free style.
+
+TODO: more detail
```

Comment:
Can you either add this, or create an issue to complete and add a link to the issue here.

Reply:
I'll open an issue for the docs as soon as the pr is merged. I plan to finish this myself, but it's gonna take some thought to come up with "real world" examples since this function came out of my own head as a solution for keeping the library small.